webapp/store/views.py

```diff
@@ -231,6 +234,37 @@ def brand_search_snap():
             status_code,
         )
 
+    # @store.route(
+    #     '/publisher/<regex("[a-z0-9-]*[a-z][a-z0-9-]*"):publisher_name>'
+    # )
+
+    def get_file(file):
+
+        try:
+            import os
+
+            with open(
+                os.path.join(flask.current_app.root_path, file), "r"
+            ) as stream:
+                data = yaml.load(stream)
+        except Exception:
+            data = None
+
+        return data
+
+    @store.route("/publisher/jetbrains")
+    def publisher_details_kde():
```

Comment:
That is not KDE anymore
```suggestion
    def publisher_details_jetbrains():
```